CaPPMS/Data/FaqManagerService.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Principal;
+using System.Threading.Tasks;
+using CaPPMS.Model;
+using Newtonsoft.Json;
+using Octokit;
```

Comment:
A good example to reinforce the comment above is this Octokit directive. It is used to interface with Github. I do not see any Github-related things here.